tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -433,7 +439,23 @@ <h3>No profile data was found.</h3>
       '_maybeUpdateActiveHosts(_selectedDatasetName, _selectedToolName)',
     ],
     attached: function() {
-      this.set('_isAttached', true);
+      // Check if the plugin was created
+      this._requestManager
+          .request(tf_backend.getRouter().pluginsListing())
+          .then(this._canceller.cancellable(result => {
+            if (result.cancelled) {
+              return;
+            }
+            if (!('profile' in result.value)) {
+              // The plugin was not created
+              this.$.initialDialog.openNoTensorFlowDialog();
+              this.set('_isAttached', false);
```

Comment:
Let's change this to `_isAvailable` like the other dashboards, but then we should also update _maybeInitializeDashboard which reads the value in two places:

- Observer registration up on line 437 should change to the following https://github.com/tensorflow/tensorboard/blob/678e6b90e0341c26691441a77c09223708012037/tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html#L437
```javascript
_maybeInitializeDashboard(_isAttached, _isAvailable)
````

- Observer definition down on line 487 should change to the following https://github.com/tensorflow/tensorboard/blob/678e6b90e0341c26691441a77c09223708012037/tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html#L487-L492

```javascript
_maybeInitializeDashboard: function(isAttached, isAvailable) {
  if (this._initialized || !isAttached || !isAvailable) {
    // Either this dashboard is already initialized ... or we are not yet
    // ready to initialize.
    return;
  }
```